crypto/cryptlib.c

```diff
@@ -272,6 +289,17 @@ void OPENSSL_showfatal(const char *fmta, ...)
     va_end(ap);
 
 # if defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0333
+#ifdef OPENSSL_SYS_WIN_CORE
+    /* ONECORE is always NONGUI and NT >= 0x0601*/
+        HRESULT tEventLog = TraceLoggingRegister(g_hProvider);
+        if (SUCCEEDED(tEventLog)) {
+            const TCHAR *pmsg = buf;
+            TraceLoggingWrite(g_hProvider, "ErrorLog",
+                TraceLoggingLevel(EVENTLOG_ERROR_TYPE),
+                TraceLoggingWideString(pmsg, "Message"));
```

Comment:
I suppose it means that there is implied dependency on UNICODE being defined. Well, nowadays it is defined unconditionally [originally it was opt-in option for 32-bit build], yet it's not really appropriate to have an implied dependency. What I'm saying is that this should either fail to compile without UNICODE being defined or it should do the right thing [and use whatever is appropriate when buf is not "wide string"].

And styling nit. That's not how we format multi-line function arguments. And this whole segment is misaligned.